libs/s25main/gameData/BuildingConsts.cpp

```diff
@@ -126,8 +129,11 @@ const helpers::EnumArray<const char*, BuildingType> BUILDING_HELP_STRINGS = {{
                "working when mines become exhausted. The "
                "desired mineral can be selected by "
                "toggling the output button."),
-  "",
-  // Nothing
+  // Tannery building
+  gettext_noop("The tanner works animals skins to form leather. "
```

Comment:
I had a typo in this part somehow: It should be "The tanner works animal skins to form leather... etc"